lib/omise/OmiseTest.php

```diff
@@ -2,13 +2,15 @@
 
 require_once dirname(__FILE__).'/res/OmiseApiResource.php';
 
-class OmiseTest extends OmiseApiResource {
-
-  /**
-   * (non-PHPdoc)
-   * @see OmiseApiResource::getInstance()
-   */
-  public static function resource() {
-    return parent::getInstance(get_class());
-  }
+class OmiseTest extends OmiseApiResource
+{
+    /**
+     * (non-PHPdoc)
+     *
+     * @see OmiseApiResource::getInstance()
+     */
+    public static function resource()
+    {
+        return parent::getInstance(get_class());
+    }
 }
```

Comment:
@guzzilar Please add a single blank line at the end of file, lib/omise/OmiseTest.php.